src/babylon.scene.ts

```diff
@@ -439,6 +439,16 @@
         * An event triggered when a mesh is removed
         */
         public onMeshRemovedObservable = new Observable<AbstractMesh>();
+        
+        /**
+        * An event triggered when a mesh is created
+        */
+        public onNewMaterialAddedObservable = new Observable<Material>();

```

Comment:
Please fix the comments :)

Reply:
done : 269c189873e036f37218eff4a2682b2d3281ab29